<launch>

    <node pkg="sensor_fusion" name="homing_update" type="homing_update" output="screen">
    </node>


</launch>
